package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "MONGODB_URI=mongodb://localhost:27017/lemme-api jest",
```

Comment:
Do we need this env? I saw that this uri is hardcoded inside `tests/globalConfig.json`